.gitignore

```diff
@@ -164,3 +164,4 @@ samples/client/petstore/typescript-node/npm/npm-debug.log
 # aspnetcore
 samples/server/petstore/aspnetcore/.vs/
 effective.pom
+\?
```

Comment:
when running run-in-docker.sh, this directory continued to get populated with jars.